### PR TITLE
Push release tags to GHCR

### PR DESCRIPTION
**Pull request by onedr0p**
Fixes: #86

### CODE REVIEW

**Comment by onedr0p** [comment]
@Rory-Z when you have a moment <3

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,6 +29,12 @@ jobs:
       with:
         username: ${{ secrets.DOCKER_HUB_USER }}
         password: ${{ secrets.DOCKER_HUB_TOKEN }}
+    - uses: docker/login-action@v2
+      if: startsWith(github.ref, 'refs/tags/')
```
Comment (Rory-Z) [inline .github/workflows/release.yml:33]: I think both of this line and 28th line can be deleted

Comment (onedr0p) [reply .github/workflows/release.yml]: Updated

Comment (onedr0p) [reply .github/workflows/release.yml]: Updated @Rory-Z

**Pull request merged**